crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/CompareContributionService.java

```diff
@@ -27,128 +25,32 @@ public class CompareContributionService {
     private final ContributionService contributionService;
 
     @Transactional
-    public int compareContribution(CalculateContributionDTO calculateContributionDTO, ContributionResult contributionResult) {
-        log.info("Start  compareContribution");
+    public boolean shouldCreateContribution(CalculateContributionDTO calculateContributionDTO, ContributionResult contributionResult) {
         int repId = calculateContributionDTO.getRepId();
         RepOrderDTO repOrderDTO = calculateContributionDTO.getRepOrderDTO();
-        List<Contribution> contributions = maatCourtDataService.findContribution(repId, false);
-        log.info(" compareContribution.contributions--" + contributions);
+        String magsCourtOutcome = calculateContributionDTO.getMagCourtOutcome() == null ? null
+                : calculateContributionDTO.getMagCourtOutcome().getOutcome();
 
+        List<Contribution> contributions = maatCourtDataService.findContribution(repId, false);
+        log.debug("shouldCreateContribution.contributions--" + contributions);
```

Comment:
Is the `log.debug` intentional here?

Reply:
Yeh it is, I thought logging the contribution data wasn't really essential unless troubleshooting an issue so I set the level to debug. Happy to change though if you think otherwise?

Reply:
That's cool, just wanted to make sure it wasn't a bit of code that was only for dev 👍